autoload/verilog_systemverilog.vim

```diff
@@ -586,7 +586,12 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
   endif
   if (l:tool == "ncverilog")
     " Based on https://github.com/vhda/verilog_systemverilog.vim/issues/88
-    set errorformat=%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat=%A%p\|,%Z%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
```

Comment:
That's a good question and I don't have an acceptable answer at present. I initially started with another project, and progressed from there (https://github.com/ryanlee/vim/blob/master/vimfiles/compiler/ncverilog.vim). 
I will remove this from the pull request as I can't re-create the condition whereby I thought it was needed.